src/main/java/com/project/mizawra/controllers/rest/RegisterRestController.java

```diff
@@ -25,10 +26,11 @@ public RegisterRestController(ApplicationEventMulticaster eventMulticaster, Clie
     }
 
     @PostMapping("/register")
-    public String registerClient(@Valid ClientDto clientDto, HttpServletRequest request) throws Exception{
+    public String registerClient(@Valid ClientDto clientDto, HttpServletRequest request, HttpServletResponse response) throws Exception{
         Client client = clientService.registerClient(clientDto);
 
         eventMulticaster.multicastEvent(new OnRegistrationCompleteEvent(client, request.getLocale()));
+        response.sendRedirect("/register/almostDone");
```

Comment:
I agree that this approach will work, but it become not rest method, because it will always give us view. So I propose to put that method to mvc controller, then it may have sense